app/src/main/res/layout/content_main.xml

```diff
@@ -10,86 +10,70 @@
 
     <Button
         android:id="@+id/button4"
-        android:layout_width="wrap_content"
+        android:layout_width="134dp"
```

Comment:
I wouldn't change from a flexible width, like wrap_content, to a fixed width, like 134dp.  134dp will not necessarily look good on all screen densities.